xml/System.Windows.Media/PathSegment.xml

```diff
@@ -29,7 +29,7 @@
 ## Remarks  
  Classes that derive from <xref:System.Windows.Media.PathSegment>, such as <xref:System.Windows.Media.ArcSegment>, <xref:System.Windows.Media.BezierSegment>, and <xref:System.Windows.Media.LineSegment>, represent specific types of geometric segments.  
   
- **Freezable Features:** Because <xref:System.Windows.Media.PathSegment> objects inherit from the <xref:System.Windows.Freezable> class, they provide several special features: they can be declared as , shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features that are provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
+ **Freezable Features:** Because <xref:System.Windows.Media.PathSegment> objects inherit from the <xref:System.Windows.Freezable> class, they provide several special features: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md),shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features that are provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```

Comment:
```suggestion
 **Freezable Features:** Because <xref:System.Windows.Media.PathSegment> objects inherit from the <xref:System.Windows.Freezable> class, they provide several special features: they can be declared as [resources](~/docs/desktop-wpf/fundamentals/xaml-resources-define.md), shared among multiple objects, made read-only to improve performance, cloned, and made thread-safe. For more information about the different features that are provided by <xref:System.Windows.Freezable> objects, see [Freezable Objects Overview](~/docs/framework/wpf/advanced/freezable-objects-overview.md).  
```